### PR TITLE
[Workers] Use suggested notation for kv_namespaces

### PR DESCRIPTION
**Pull request by emielvanseveren**
implementation of https://github.com/cloudflare/cloudflare-docs/pull/4576

### CODE REVIEW

**Comment by koeninger** [comment]
if we're changing kv examples to use the double bracket syntax, can you add a second example namespace for the benefit of people who aren't familiar with toml?

**Comment by emielvanseveren** [comment]
@koeninger, sure, what about r2 buckets? Personally I'd change the default syntax as well, it decreases the chance of similar problems.

**Comment by koeninger** [comment]
I can't speak for the R2 team, but I agree it seems less error prone

**Review by koeninger — APPROVED** [review]

File: content/workers/wrangler/configuration.md [hunk 1]
```diff
@@ -72,9 +72,15 @@ KEY = "value"
 # @param {string} id The ID of the KV namespace at the edge
 # @param {string} preview_id The ID of the KV namespace used during `wrangler dev`
 # not inherited
-kv_namespaces = [
-  { binding = "TEST_NAMESPACE", id = "", preview_id = "" }
-]
+[[kv_namespaces]]
+  binding = "TEST_NAMESPACE_0"
+  id = ""
+  preview_id = ""
```
Comment (deadlypants1973) [inline content/workers/wrangler/configuration.md]:
```suggestion
  binding = "<YOUR_NAMESPACE_0>"
  id = "<YOUR_ID>"
  preview_id = "<YOUR_PREVIEW_ID>"
```

File: content/workers/wrangler/configuration.md [hunk 2]
```diff
@@ -72,9 +72,15 @@ KEY = "value"
...
+
+[[kv_namespaces]]
+  binding = "TEST_NAMESPACE_1"
+  id = ""
+  preview_id = ""
```
Comment (deadlypants1973) [inline content/workers/wrangler/configuration.md]:
```suggestion
  binding = "<YOUR_NAMESPACE_1>"
  id = "<YOUR_ID>"
  preview_id = "<YOUR_PREVIEW_ID>"
```

**Review by deadlypants1973 — CHANGES_REQUESTED** [review]

File: content/workers/wrangler/configuration.md [hunk 3]
```diff
@@ -72,9 +72,15 @@ KEY = "value"
 # @param {string} id The ID of the KV namespace at the edge
 # @param {string} preview_id The ID of the KV namespace used during `wrangler dev`
 # not inherited
-kv_namespaces = [
-  { binding = "TEST_NAMESPACE", id = "", preview_id = "" }
-]
+[[kv_namespaces]]
+  binding = "TEST_NAMESPACE_0"
```
Comment (kristianfreeman) [inline content/workers/wrangler/configuration.md]: Do these need to be space-indented like this?

**Review by deadlypants1973 — APPROVED** [review]

**Pull request merged**